tests/test_basic.py

```diff
@@ -5,11 +5,67 @@
 import unittest
 
 
+import crash_similarity
+
+
 class BasicTest(unittest.TestCase):
 
-    def test_pass(self):
+    # Train Model to be used in all tests
```

Comment:
You can use a setup function to perform these initializations: https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUp